tests/unit/htdocs/class/file/XoopsFileHandlerTest.php

```
@@ -0,0 +1,1132 @@
+        $handler = new XoopsFileHandler($this->tempFile);
+
+        $input = "line1\nline2\r\nline3\rline4";
+        $result = $handler->prepare($input);
+
+        // On Windows (which is the test environment), all line breaks
+        // should be converted to \r\n
+        if (strncasecmp(PHP_OS, 'WIN', 3) === 0) {
+            $expected = "line1\r\nline2\r\nline3\r\nline4";
+        } else {
+            $expected = "line1\nline2\nline3\nline4";
+        }
```

PHPMD (2.15.0)

[error] The method testPrepareNormalizesLineEndings uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)